File: Python/lt_01_estseq/lt01_estseq_36.py
def ffatorial(num) -> int:
    fat = 1
    for i in range(1, num + 1, 1):
        fat *= i
    return fat


def fserie(num) -> float:
    soma = 1
    if num > 1:
        for i in range(1, num + 1, 1):
            soma += 1 / ffatorial(i)
    return soma


num = int(input("Insira o numero de Elementos da Série:"))
print("O valor da Série com", num, "valores é:", fserie(num))

File: C/lt_01_estseq/lt01_estseq_36.c
#include <stdio.h>

int fat(int *num) {
  int fat = *num, i;
  for (i = *num - 1; i > 0; i--) {
    fat *= i;
  }
  return fat;
}

double serie(int *num) {
  double serie = 0;
  int i;

  for (i = 1; i <= *num; i++) {
    serie += 1.0 / fat(&i);
  }
  return serie;
}

int main() {
  int num;

  printf("Insira um numeor a ser calculada a serie: ");
  scanf("%d", &num);

  printf("O valor da serie será %.2lf.\n", serie(&num));
  return 0;
}
